client/src/components/GraphViewerComponent/GraphViewerComponent.js

```diff
@@ -245,7 +246,7 @@ class GraphViewerComponent extends React.Component {
             this.setState({ couldNotDisplay: true });
           }
         } else if (data.relationships.length > 0) {
-          this.setState({ couldNotDisplay: true, relationshipsOnly: true });
+          this.setState({ couldNotDisplay: true, relationshipsOnly: true, relationships: data.relationships });
```

Comment:
I don't think we need to put the rels in state as they don't impact rendering?

Could we put them in `this.relationships`? That seems to be an existing property.

Reply:
Changed into this.relationships.